utils/qemu/Makefile

```diff
@@ -9,10 +9,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=qemu
-PKG_VERSION:=5.0.0
-PKG_RELEASE:=5
+PKG_VERSION:=6.0.0
+PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_HASH:=2f13a92a0fa5c8b69ff0796b59b86b080bbb92ebad5d301a7724dd06b5e78cb6
+PKG_HASH:=87bc1a471ca24b97e7005711066007d443423d19aacda3d442558ae032fa30b9
 PKG_SOURCE_URL:=http://download.qemu.org/
 PKG_LICENSE:=GPL-2.0
```

Comment:
This should be either -only or -or-later, as per SPDX.

Reply:
I'm not quite sure what should be used: https://github.com/qemu/qemu/blob/master/LICENSE

It sounds to me, that most of the code is GPL-2.0-only, but some parts could be GPL-2.0-or-later.

Reply:
Use -only then.

Reply:
Done.